fix: hard-code operator fingerprint

mook-as opened the pull request:
Addresses https://github.com/cloudfoundry-incubator/kubecf/pull/1275#discussion_r489395441

mook-as committed:
fix: hard-code operator fingerprint

Addresses
https://github.com/cloudfoundry-incubator/kubecf/pull/1275#discussion_r489395441

f0rmiga commented on main.go:
```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/SUSE/eirini-dns-aliases/pkg/extension"
 )
 
+const (
+	operatorFingerprint = "eirini-dns-aliases"
+)
```
```suggestion
const operatorFingerprint = "eirini-dns-aliases"
```

mook-as committed:
fix: collapse const block

Accepts https://github.com/SUSE/eirini-dns-aliases/pull/5#discussion_r489612948

Co-authored-by: Thulio Ferraz Assis <thulio.assis@suse.com>
